components/com_content/src/Model/ArticleModel.php

```diff
@@ -450,6 +450,54 @@ public function storeVote($pk = 0, $rate = 0)
 		return false;
 	}
 
+	/**
+	 * Save Title of an article
+	 *
+	 * @param   integer $pk     Article id
+	 *
+	 * @param   string  $title  New title of the article
+	 *
+	 * @return  boolean True if successful, false otherwise
+	 *
+	 * @since ___add_version___
+	 */
+	public function saveTitle($pk = null, $title = null)
```

Comment:
Specify the argument type for the parameters and return type
```php
public function saveTitle(int $pk = null, string $title = null): void
```

I have set the return type to void as I am op the opinion that we should not have to return any value. Instead of there is a problem throw an exception.